chromium/browser_type.go

```diff
@@ -334,6 +334,7 @@ func parseArgs(flags map[string]any) ([]string, error) {
 func prepareFlags(lopts *common.BrowserOptions, k6opts *k6lib.Options) (map[string]any, error) {
 	// After Puppeteer's and Playwright's default behavior.
 	f := map[string]any{
+		"disable-gpu":                                        true, // fixes CI tests.
```

Comment:
Out of curiosity, how did you get to this?

Reply:
TBH, I just experimented with it when I see it [here](https://bugs.chromium.org/p/chromium/issues/detail?id=383314), and I don't have the best answer.